src/ds5/ds5-wide-fov.cpp

```diff
@@ -0,0 +1,21 @@
+#include "ds5-wide-fov.h"
+
+namespace librealsense
+{
+    std::vector<tagged_profile> ds5_wide_fov::get_profiles_tags() const
+    {
+        std::vector<tagged_profile> tags;
+        auto usb_spec = get_usb_spec();
+        if (usb_spec >= platform::usb3_type || usb_spec == platform::usb_undefined)
```

Comment:
You also can use the `ds5-factory.cpp` method for maintainability and clearness:
```c++
 bool usb3mode = (usb_spec >= platform::usb3_type || usb_spec == platform::usb_undefined);
uint32_t width  = usb3mode ?  1280 : 640;
uint32_t height = usb3mode ?   720 : 480;
uint32_t fps    = usb3mode ?    30 :  15;

tags.push_back({ RS2_STREAM_COLOR, -1, width, height, RS2_FORMAT_RGB8, fps,...
```